src/Nethermind/Nethermind.Merge.Plugin/Synchronization/BeaconHeadersSyncFeed.cs

```diff
@@ -111,22 +121,22 @@ protected override AddBlockResult InsertToBlockTree(BlockHeader header)
             // if (_blockTree.LowestInsertedHeader != null
             //     && _blockTree.LowestInsertedHeader.Number < (_blockTree.LowestInsertedBeaconHeader?.Number ?? long.MaxValue))
```

Comment:
hmmm do you think we can revert this if comment?

Reply:
we can revert in another branch